features/raadpleeg-bewoning-met-periode/test/naam.feature

```diff
@@ -0,0 +1,102 @@
+            #language: nl
+
+            @api
+            Functionaliteit: raadpleeg bewoning in periode levert naam bewoner
+
+            Als consumer van de Bewoning API
+            wil ik kunnen opvragen welke personen in een periode op een adresseerbaar object verblijven/hebben verbleven
+            zodat ik deze informatie kan gebruiken in mijn proces
+
+            Achtergrond:
+            Gegeven adres 'A1' heeft de volgende gegevens
+            | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
+            | 0800                 | 0800000000000001                         |
+            En adres 'A2' heeft de volgende gegevens
+            | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
+            | 0800                 | 0800000000000002                         |
+
+    Regel: een persoon is binnen een periode bewoner van een adresseerbaar object als:
+    - de van datum van de periode valt op of na datum aanvang adreshouding van de persoon op het adresseerbaar object
+    - de tot datum van de periode valt vóór datum aanvang adreshouding van de persoon op het volgende adresseerbaar object
+
+    Scenario: bewoning wordt gevraagd voor een periode die ligt binnen de verblijfperiode van één persoon op het adresseerbaar object en persoonsgegevens zijn in onderzoek
+            Gegeven de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
+            | voornamen (02.10) | adellijke titel of predicaat (02.20) | voorvoegsel (02.30) | geslachtsnaam (02.40) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) |
```

Comment:
Cathy heeft aan het eind van de ochtend doorgegeven dat het niet meer nodig is om 'in onderzoek' gegevens te leveren